server/cp/output/formatter/cp_ninjs_newsroom_formatter.py

```diff
@@ -9,16 +9,17 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 
-from .ninjs_formatter import NINJSFormatter

```

Comment:
hi @cpefatimaabdillahi this is the import I'm not sure about, because there is no such file